ui/js/controllers/perf/alerts.js

```diff
@@ -12,10 +12,29 @@ perf.factory('PhBugs', [
                         return alert.status !== phAlertStatusMap.INVALID;
                     });
                     var template = response.data[0];
+                    function getAlertSummary(alertSummary) {
+                        var resultStr = "";
+                        var improved = _.filter(alertSummary.alerts, function(alert) {return !alert.is_regression && alert.visible;});
+                        var regressed = _.filter(alertSummary.alerts, function(alert) {return alert.is_regression && alert.visible;});
+                        if (improved.length > 0) {
+                            resultStr += "Summary of tests that improved\n" +
+                                         _.map(improved, function(alert) {
+                                             return alert.title + " - " + alert.amount_pct + "% better";
+                                         }).join('\n') + "\n";
+                        }
+                        if (regressed.length > 0) {
+                            resultStr += "Summary of tests that regressed\n" +
+                                         _.map(regressed, function(alert) {
+                                             return alert.title + " - " + alert.amount_pct + "% worse";
+                                         }).join('\n') + "\n";
```

Comment:
Same general changes described above for improvements should be applied here. :)